src/types/globals/Window.js

```diff
@@ -7,13 +7,13 @@ import type { IDBFactory, IDBKeyRange } from '../indexeddb';
 import type { SymbolTableAsTuple } from '../../profile-logic/symbol-store-db';
 import type { GoogleAnalytics } from '../../utils/analytics';
 
-declare class GeckoProfiler {
+declare type GeckoProfiler = {
```

Comment:
I needed structural typing to mock the symbol table, instead of nominal typing.

Reply:
I believe an interface was more correct than a class, and with an interface you could have "implemented" it. But I don't mind having a type instead :)